presto-main/src/main/java/com/facebook/presto/server/TaskInfoResource.java

```diff
@@ -69,4 +118,34 @@ public TaskInfo getTaskInfo(@PathParam("taskId") TaskId taskId)
             throw new NotFoundException(e);
         }
     }
+
+    private void proxyTaskInfo(HttpServletRequest servletRequest, AsyncResponse asyncResponse, String xForwardedProto, UriInfo uriInfo)
+    {
+        try {
+            checkState(proxyHelper.isPresent());
+            Optional<InternalNode> resourceManager = internalNodeManager.getResourceManagers().stream()
+                    .findFirst();
+            if (resourceManager.isPresent()) {
+                InternalNode resourceManagerNode = resourceManager.get();
+                URI uri = uriInfo.getRequestUriBuilder()
+                        .queryParam("includeLocalQueryOnly", true)
```

Comment:
"includeLocalQueryOnly" is a magic string. Can we have a constant var defined somewhere for this.

Reply:
Replaced with constant `INCLUDE_LOCAL_QUERY_ONLY`